extension/src/project/projectCommands.ts

```diff
@@ -99,6 +100,45 @@ export function registerProjectCommands(context: vscode.ExtensionContext) {
             }
         }));
 
+        context.subscriptions.push(vscode.commands.registerCommand('autolisp.addWorkspaceFile2Project', async (clickedFile: vscode.Uri, selectedFiles: vscode.Uri[]) => {
```

Comment:
And I see in the project root node to right click there is also an menu "add file to project", for this menu it is better that it can also support to select a folder.

Reply:
You don't think the ability to use standard multi-selection is sufficient? Does VSCode even have a decent folder picker? I feel like that's a pretty large deficiency existing in all newer Microsoft related frameworks.